addons/payment/tests/test_payment_token.py

```diff
@@ -0,0 +1,22 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from freezegun import freeze_time
+
+from odoo.tests import tagged
+
+from odoo.addons.payment.tests.common import PaymentCommon
+
+@freeze_time('2022-07-07')
+@tagged('-at_install', 'post_install')
+class TestPaymentToken(PaymentCommon):
+
+    def test_token_default_display_name(self):
+        token = self._create_token(payment_details='1234')
```

Comment:
In the `_create_token` method, you have a default value for the `payment_details`, why not using it?

Unless you are specifically testing something with a specific `payment_details` to test a different comportment (as it's the case just after), going with the default value is fine